### ISSUE TITLE
Bootstrap problems with react-native-vanilla

### ISSUE THREAD

**Issue by shilman** [storybooks/storybook#1509]
## why

The new build setup (I think?) is breaking `examples/react-native-vanilla`.

## what

Here's my setup (on `master`):
```
git clean -fdx && yarn && yarn bootstrap && yarn bootstrap:react-native-vanilla
```

This causes a few problems:
1. Primarily, the example doesn't run: it's hitting this error `Accessing nonexistent addons channel, see https://storybook.js.org/basics/faq/#why-is-there-no-addons-channel`
   - I've tried changing this from an error to a warning with `console.log`, but it doesn't help
   - I've tried increasing the `setTimeout` for addons to 1000 just in case
2. Secondarily, the bootstrap procedure only works when I do a clean build. When I modify a file outside the `react-native-vanilla` directory and re-bootstrap it gives me the following error. The only way to fix it is to run `git clean -fdx` but this is slow and makes RN development impractical:

```
MBP:react-native-vanilla shilman$ cd .. && yarn bootstrap && yarn bootstrap:react-native-vanilla && cd - && yarn storybook
yarn storybook v0.27.5
$ storybook start -p 7007
sh: storybook: command not found
error Command failed with exit code 127.
```

**Comment by shilman** [comment]
NOTE: I have just tried `v3.1.9` on a standalone RN project and verified that this is a development problem and not a problem with our latest public release.

**Comment by shilman** [comment]
@ndelangen @tmeasday could use your help on this 😉

**Comment by shilman** [comment]
Mediocre workaround for problem #2:
```
rm -rf node_modules && cd .. && yarn bootstrap && yarn boostrap:react-native-vanilla && cd - && yarn storybook
```

**Comment by ndelangen** [comment]
Hey @shilman let's sit together and help me get setup to help you fix this, OK? When's a good time for you?

**Comment by tmeasday** [comment]
@shilman: I thought we determined that RN didn't work with symlinks unless we used that other packager?

**Comment by shilman** [comment]
@tmeasday fixed a couple problems, see PR #1514 above

**Comment by danielduan** [comment]
Think this has been fixed.
